toqito/matrix_ops/commutant.py

```diff
@@ -0,0 +1,59 @@
+"""Module for computing the commutant of a set of matrices."""
+
+import numpy as np
+from scipy.linalg import null_space
+
+
+def commutant(A: np.ndarray | list[np.ndarray]) -> list[np.ndarray]:
+    r"""Compute an orthonormal basis (in the Hilbert-Schmidt inner product).
+
+    For the algebra of matrices that commute with each matrix in A.
+
```

Comment:
This comment seems a bit strange on its own like this. Is there a way to make this more complete?